extensions/logging/README.md

```diff
@@ -23,6 +23,8 @@ npm install --save @loopback/logging
 
 ## Basic use
 
+{% include note.html content="this.configure() must be called before this.component() to take effect." %}
```

Comment:
I think this might be better put under the "Configure the logging component" section since the reader will have more context at that point. And instead of a note, it could be a sentence since people reading this on GitHub/npmjs won't see the note formatting. What do you think?

Reply:
This was lifted from the [health extension README](https://github.com/strongloop/loopback-next/blame/883e9b31b0e5281085a0c9f091f648e84fef795e/extensions/health/README.md#L14-L15). Since the logging extension README is also [rendered in the docs website](https://github.com/strongloop/loopback-next/blob/883e9b31b0e5281085a0c9f091f648e84fef795e/docs/site/Logging.md), I thought it'd be good to keep the note consistent.

If we're going to alter the note for logging, we should also consider changing the note on health to keep it consistent.

Reply:
My only concern with using notes in README is how they get formatted (or rather not formatted) outside the website (e.g. https://www.npmjs.com/package/@loopback/health), but on the other hand, I think it should be okay since I'm assuming most people will be reading it from the docs website.

I still do think we can alter where it is in terms of where it is in the README though for both here and the health extension so the reader has more context before reading the note.

Reply:
I think that's a good point; I've shifted the health and logging note next to the configuration code snippet.